addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1398,13 +1399,14 @@ export class Wysiwyg extends Component {
                         data.rel = 'ugc';
                     }
                     data.linkDialog.applyLinkToDom(data);
+                    this.odooEditor.historyUnpauseSteps();
```

Comment:
Hello, @Jinjiu96 just a small concern with this change here i have in mind. What i think `this.odooEditor.historyUnpauseSteps()` here will do is it will make link undo itself although the document has been saved. I dont think we should be undoing stuff after the document has been saved. I have checked in previous version the link dont undo itself after being saved. Please share your view cc @dmo-odoo

Reply:
@shsa-odoo This is not meant to revert the link but to counteract that line: https://github.com/odoo/odoo/pull/138517/files#diff-aa4686cd9f6a0d50e1040ba3dda0b29e9898e7cc31a47f1f02b46a2c4c047a72R1377

Do you have steps to reproduce the issue you are talking about reverting the link ?

Reply:
@shsa-odoo hello, thanks for checking! This step is added to have the correct setup in the `historyStep`, when `historyUnpauseSteps` is called, it simply set `_historyStepsActive` as true. This is necessary cause `historyStep` use this variable to set up the next history step correctly. It doesn't mean we will remove the link but more an operation on the recorded mutations if I understand correctly.

The part in `historyStep` is below:
https://github.com/odoo/odoo/blob/4b6756e1775488e230bc25767adf5744f0ea9149/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L1189-L1193

Reply:
@Jinjiu96 , @dmo-odoo  the issue is not with the definition of it, but my question is why i am able to undo the change made (here created link) after saving the document ? which in my view should not happen because the document got saved. Please try it yourself what i am trying to say,

steps to reproduce:
1) open new note
2) type "hello" select it and convert it to link
3) save the note
4) again click on note to edit it.
5) hit ctrl+z it will undo the link created.

is this intended ? because when i comment that line, the link doesn't gets undone after saving . i will post a video in task as well.

Reply:
Hello, @dmo-odoo, after a discussion with @shsa-odoo , we found it's only reproducible when creating a new note from the dashboard. The same issue exists in the previous versions so we will handle it in another task